frontend/src/hocs/withHostCommunication/withHostCommunication.test.tsx

```diff
@@ -80,11 +80,26 @@ describe("withHostCommunication HOC", () => {
 
 describe("withHostCommunication HOC receiving messages", () => {
   let dispatchEvent: any
+  let originalHash: any
   let wrapper: any
+  let hostCommunication: any
 
   beforeEach(() => {
+    originalHash = window.location.hash
```

Comment:
Why is this saved and restored? Does it warrant a comment? (If it's a common frontend test pattern that I'm just ignorant of, no need to add one!)

Reply:
It's not uncommon when dealing with the `window` object in tests but probably still warrants an explanatory comment because it's not immediately obvious why this is necessary and comes up relatively infrequently.